app/client/components/RegionFocusSwitcher.ts

```diff
@@ -481,8 +482,18 @@ const focusPanel = (panel: PanelRegion, child: HTMLElement | null, gristDoc: Gri
   }
 };
 
+const focusPanelElement = (panelElement: HTMLElement) => {
+  // tabindex is set here and removed later with removeTabIndexes(), instead of
+  // directly set on the element on creation, for a reason:
+  // if we happen to just click on a non-focusable element inside a panel,
+  // browser default behavior is to make document.activeElement the closest focusable parent (the panel).
```

Comment:
Is that true across browsers? I wasn’t aware of this (or never really realised it in these terms maybe).

Reply:
I didn't test _everything_ but I have this behavior on Firefox and Chrome (on Linux). [Here is a test page](https://659dhq.csb.app/).